README.md

```diff
@@ -108,25 +108,12 @@ import (
 ...
 ```
 
-### Pinning to the `v1` releases
+### Using `v1` releases
 
-Similarly to the section above describing use of the `v2` branch, if one wants
-to avoid any unexpected compatibility pains once `v2` becomes `master`, then
-pinning to `v1` is an acceptable option, e.g.:
+Use the github URL to use the v1 branch.  This is illustrated in the Getting Started section below.
```

Comment:
```suggestion
$ go get github.com/urfave/cli

package main

import (
  "github.com/urfave/cli"
)
```

I know its _right below_, but for some things (like install directions) its fine to copy paste IMO.